gshoppingflux/gshoppingflux.php

```diff
@@ -1634,7 +1634,7 @@ public function renderInfo()
 		));
 	}
 
-	public function customGetNestedCategories($shop_id, $root_category = null, $id_lang = false, $active = true, $groups = null, $use_shop_restriction = true, $sql_filter = '', $sql_sort = '', $sql_limit = '')
```

Comment:
Maybe a better way to get rid of the exception could be to simply remove the semicolon on line 1741? By this way it's still possible to test different filters afterwards, if needed. What do you think?